onedal/dal.cpp

```diff
@@ -15,6 +15,7 @@
 *******************************************************************************/
 
 #include "onedal/common.hpp"
+#include "onedal/version.hpp"
```

Comment:
Why not use `services/library_version_info.h`?

Reply:
For me these macros like __INTEL_DAAL_XYZ__ look weird and outdated. We developing a new part of the library and the old one receives little to no updates.